rebasehelper/application.py

```diff
@@ -142,8 +142,7 @@ def _prepare_spec_objects(self):
             logger.debug("argument passed as a new source is a version")
             version, extra_version = SpecFile.split_version_string(self.conf.sources)
             self.rebase_spec_file.set_version(version)
-            if extra_version:
-                self.rebase_spec_file.set_extra_version(extra_version)
+            self.rebase_spec_file.set_extra_version(extra_version)
```

Comment:
It will work, there is another check for empty extra_version in set_extra_version method.
Also, when archive name is passed as an argument to rebase-helper and set_extra_version is called from set_version_using_archive method, there is no check for empty extra_version there either ([code](https://github.com/phracek/rebase-helper/blob/3820af9ffe8ff62e76d3afed467bd52d06860fd3/rebasehelper/specfile.py#L858)).